src/ecs/Entity.ts

```diff
@@ -1,9 +1,9 @@
-import {getComponentClass, getComponentId} from './ComponentId';
-import {Class} from '../utils/Class';
-import {Signal} from '../utils/Signal';
-import {isTag, Tag} from './Tag';
-import {ILinkedComponent, isLinkedComponent} from './LinkedComponent';
-import {LinkedComponentList} from './LinkedComponentList';
+import { getComponentClass, getComponentId } from './ComponentId';
```

Comment:
Please revert these changes in import formatting, as they are related to your local formatting settings. But irrelevant for the goal of the pull-request.